### ISSUE TITLE
Build and Run in term is very slow

### ISSUE THREAD

**Issue by daxiawj** [textmate/fortran.tmbundle#13]
When I Compile and Execute Single "Hello, world" fortran file use ⌘+R,  it will take a long time (more than 1 minute) to response and run the terminal to display the result.  While if I just Compile it ⌘+B, it will response quickly.

Would you please help? My TextMate is 2.0-alpha.9497.

**Comment by infininight** [comment]
The main difference in the two commands is that one uses AppleScript to run the resulting compiled output. AppleScript does have some startup lag but it shouldn't be longer than a few seconds.

The only thing I can think of is that Applescript is taking too long to find Terminal or TextMate for compiling the script, are you having any issues with application launching or such?

**Comment by daxiawj** [comment]
No, I don't have such problem. My SSD drive works well, and every application launches very quick either from Spotlight or double click in /Application folder. 

I think I may already figured out the issue. my iTerm is iTerm2, and I modified the original Applescript code to following:

``` applescript
osascript <<EOF
tell application "iTerm"
    make new terminal
    tell the current terminal
        activate current session
        launch session "Default Session"
        tell the last session
            # write text "cd '$TM_DIRECTORY'; ./${TM_FILENAME%.*}"
             write text "cd '$TM_DIRECTORY'; ./a.out"
        end tell
    end tell
end tell
EOF
```

and Problem solved. Thanks!
